src/block-support/supported-blocks.json

```diff
@@ -29,10 +29,12 @@
 		"core/file",
 		"core/search",
 		"core/block",
+		"jetpack/address",
 		"jetpack/contact-info",
 		"jetpack/email",
+		"jetpack/layout-grid",
 		"jetpack/phone",
-		"jetpack/address"
+		"jetpack/story"
```

Comment:
I _think_ this should be fine, but let's cc: @antonis too since Chip isn't on this project anymore.

Reply:
I'll still bounce in. All of the layouts are also tagged on the backend to be marked to have them included so if there is an issue we could remove that flag. However worth noting is this issue I raised. https://github.com/wordpress-mobile/gutenberg-mobile/issues/3804

The Jetpack blocks aren't be stripped out for sites that don't support JetPack. So it would actually be safer to not include them until that list matches the site's capabilities. But only safer in terms of if we want to expose those layouts to users.

Reply:
Thanks for the feedback, everyone. 🙏

- [ ] Revert this list back since it sounds like it may be safer. (I was unsure exactly how to check when adding them to the supported list originally.)

Reply:
Done.

Admittedly, this part wasn't completely clear to me.
> But only safer in terms of if we want to expose those layouts to users.